MQTTClient/MQTTClient/SRWebSocket.m

```diff
@@ -1482,7 +1485,8 @@ - (void)_sendFrameWithOpcode:(SROpCode)opcode data:(id)data;
         }
     } else {
         uint8_t *mask_key = frame_buffer + frame_buffer_size;
-        SecRandomCopyBytes(kSecRandomDefault, sizeof(uint32_t), (uint8_t *)mask_key);
+        int ignore = SecRandomCopyBytes(kSecRandomDefault, sizeof(uint32_t), (uint8_t *)mask_key);
+        ignore = ignore;
```

Comment:
`ignore = ignore; `?

Reply:
Yes, this line avoids the unused variable error when we are generating the xcframework.

Reply:
But if the variable is not unused, why do we need it then? Before there was no variable.

Reply:
This is an old code. And if you try to compile xcframework the compiler launches an error because SecRandomCopy returns anything but we don't have any variable. So If you put some variable then the compiler launches another error because this variable is not used. Before this PR we don't needed compile this project as xcframework, so we don't see this problem before. The best correct way is doing some wrapper to ignore this variable using a macro but I think that this wrapper isn't part of this PR. Finally, we want to change this third-party dependency in a near future so I think that doesn't not worth any change here in a near future.